app/controllers/api/v1/organizations_controller.rb

```diff
@@ -1,22 +1,19 @@
 module Api::V1
   class OrganizationsController < ApiBaseController
-    before_action :set_organization, only: [:show, :update, :grant_permission, :admins, :destroy]
```

Comment:
Does this remove the ability to delete organizations?

Reply:
We decided that we weren't going to allow organizations or programs to be deleted at the last Monday meeting. That action is unreachable due to how the routes are set up anyway. For our own capabilities, we can just use the console if we really want to delete an organization or program, but if you think there is a case to leave this open for the super_admin, let me know. If so, the routes would have to be updated as well. But there are no plans to utilize this action from the FE at this time.

Reply:
I might have misunderstood the final decision during the meeting. It still seems like this basic functionality should be supported, but if it won't be available in the FE, then it probably makes sense to leave it out.

Reply:
Well, the only way to really "support it" is to have a fully functioning route and action for it, at which point we're back to making the same decision about what to do with associated programs when an organization is deleted.